spec/features/users/regular_user/profile_order_spec.rb

```diff
@@ -24,7 +24,21 @@
   end
 
   it "see a flash message confirming my recent order and empty cart" do
-    visit "profile/orders"
+    visit item_path(@item_1)
+    click_on "Add To Cart"
+
+    visit "/cart"
+    click_on "Checkout"
+
+    fill_in :name, with: "Bert"
```

Comment:
In order to hit #create method in orders controller, we had to manually fill in the new order form, as opposed to using a factory-bot object :O